### PR TITLE
Github templates added

### PR DESCRIPTION
**Pull request by dcuestam**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Contributing
+
+Contributions are welcome and are greatly appreciated! Every
+little bit helps, and credit will always be given.
+
+
```
Comment (avilchess) [inline CONTRIBUTING.md]: Delete one blank line

File: README.md [hunk 2]
```diff
@@ -33,27 +33,8 @@ The documentation can be consulted in 2 ways:
...
-`git checkout -b feature/example_feature`
-3. Develop the new feature on the the new branch. It includes testing and documentation.
-`git commit -a -m "Bla, Bla, Bla";  git push`
-4. Open a Pull Request to merge the feature branch in to develop. Currently, a pull request has to be reviewed at least by one person.
-5. Finally, delete the feature branch.
-6. Move back to develop branch.
-`git checkout develop`
-7. Pull the latest changes.
-`git pull`
+### Contributing
```
Comment (otorreno) [inline README.md]: 2 '#'

**Review by otorreno — CHANGES_REQUESTED** [review]
Take a look at the comments

**Review by otorreno — APPROVED** [review]
LGTM

**Pull request merged**